crates/criticalup-core/src/config/paths.rs

```diff
@@ -74,11 +85,50 @@ fn find_cache_dir(whitelabel: &WhitelabelConfig) -> Option<PathBuf> {
 fn platform_specific_cache_dir(whitelabel: &WhitelabelConfig) -> Option<PathBuf> {
     dirs::cache_dir().map(|v| v.join(whitelabel.name))
 }
+fn try_migrating_deprecated_path(
+    deprecated_product_name_path: PathBuf,
+    new_products_path: PathBuf,
+) -> Result<(), Error> {
+    let new_product_name_path = new_products_path.join("ferrocene");
+
+    // This function should error if we try to overwrite an existing cache
+    if matches!(new_product_name_path.try_exists(), Ok(true)) {
+        return Err(Error::CouldNotTransferCache {
```

Comment:
```suggestion
        return Err(Error::CouldNotMigrateCache {
```
lets keep the same name around `migrate`